adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1335,6 +1398,36 @@ abstract class AvroReadGroupGenomicRDD[T <% IndexedRecord: Manifest, U <: Produc
    */
```

Comment:
Slightly unrelated, would it make sense to rename `AvroReadGroupGenomicRDD` &rarr; `AvroRecordGroupGenomicRDD`?